ui/settings_page.go

```diff
@@ -12,15 +12,20 @@ import (
 
 const PageSettings = "Settings"
 
+type optionRow struct {
+	clickable *widget.Clickable
+	icon      decredmaterial.IconButton
```

Comment:
Having an clickable widget and an icon button do the same thing seems redundant. The clickable widget and an icon might just be all that's needed. Reassess the code and see if you really need to create a new struct type for this.